README.md

```diff
@@ -26,16 +26,15 @@ We can discuss the process in this [discussion thread](https://github.com/huggin
 
 This course provides a practical, hands-on approach to working with small language models, from initial training through to production deployment.
 
-| Module | Description | Status | Release Date |
-|--------|-------------|---------|--------------|
-| [Instruction Tuning](./1_instruction_tuning) | Learn supervised fine-tuning, chat templating, and basic instruction following | ✅ Ready | Dec 3, 2024 |
-| [Preference Alignment](./2_preference_alignment) | Explore DPO and ORPO techniques for aligning models with human preferences | ✅ Ready  | Dec 6, 2024 |
-| [Parameter-efficient Fine-tuning](./3_parameter_efficient_finetuning) | Learn LoRA, prompt tuning, and efficient adaptation methods | ✅ Ready | Dec 9, 2024 |
-| [Evaluation](./4_evaluation) | Use automatic benchmarks and create custom domain evaluations | ✅ Ready | Dec 13, 2024 |
-| [Vision-language Models](./5_vision_language_models) | Adapt multimodal models for vision-language tasks | ✅ Ready | Dec 16, 2024 |
-| [Synthetic Datasets](./6_synthetic_datasets) | Create and validate synthetic datasets for training | ✅ Ready | Dec 20, 2024 |
-| [Inference](./7_inference) | Infer with models efficiently | ✅ Ready | Jan 8, 2025 |
-| [Agents](./8_agents) | Build your own agentic AI | ✅ Ready | Jan 13, 2025 ||
+| Module | Description | Status | Release |
```

Comment:
2025 for the release dates? 😄 
Additionally, I find it confusing that modules point to old material so maybe a warning could be added. For example for the first one, it's not point to units/en/unit1.
`1_instruction_tuning` folder could be removed?

Reply:
SmolLM2 is still mentioned in README!